### PR TITLE
Update the apt cache before the first package install

### PR DESCRIPTION
**Pull request by smacz42**
When running simply this role on a DigitalOcean droplet, the role failed
with:

```
FAILED! => {"changed": false, "msg": "No package matching 'python3-pip'
    is available"}""}
```

After adding this, that error no longer appears and the role completes
successfully. This is akin to the start of the travis tests:

https://github.com/nickjj/ansible-docker/blob/master/tests/test.yml#L38-L41

### CODE REVIEW

**Comment by nickjj** [comment]
Hi,

Thanks for the contribution but doing an apt-get update isn't really something that should happen at the individual role level because if you go down that route then all of your roles need this -- but that'll require spending a tremendous amount of time unnecessarily running apt get updates or having to configure each role with a cache timeout (I used to do this in the past and it was a mess).

This has been discussed before at: https://github.com/nickjj/ansible-docker/issues/37

Basically, things like enabling third party apt modules and running an initial apt-get update should really happen in a designated bootstrap / apt role. The reason the update is run in the tests is so that the tests can be run without depending on a third party role.

**Comment by smacz42** [comment]
Thanks for sharing your experience. That rationale makes sense. Thanks!
